### DISCUSSION TITLE
[Suggestion] DPS Counter - sort by damage/highlight specific player

### DISCUSSION THREAD

**Discussion by ToiletBowl** [runelite/runelite#15246] [Ideas]
Default sort damage done by party members high to low, current sort is by player name. 

Add a text box to that allows the ability to highlight specific players name in the damage overlay.

**Comment by Hanssssssssssss** [comment]
bump

**Comment by JoeOSRS** [comment]
Bump, didn't this guy do it https://github.com/runelite/runelite/pull/15517 ?
